.happy/terraform/modules/swipe-sfn/sfn.yml

```diff
@@ -43,33 +43,29 @@ States:
           - Name: SFN_CURRENT_STATE
             Value.$: $$.State.Name
     ResultPath: $.BatchJobDetails.Run
-    Next: RunSPOTReadOutput
+    Next: RunReadOutput
     Retry: &BatchRetryConfig
       - ErrorEquals: ["Batch.AWSBatchException"]
         IntervalSeconds: 15
         MaxAttempts: 3
+        BackoffRate: 2.0
```

Comment:
what is `BackoffRate` used for?

Reply:
This means to double the wait interval between each backup. This is helpful if failures happen due to some sort of rate limiting since it will dynamically slow the total rate to fit the limit. This value is actually the default I just realized but it was set on our end and I am syncing them so I ported this too even though I guess it is redundant. At least it makes it more explicit what the rate is to the reader.